src/Parsers/ASTIdentifier.cpp

```diff
@@ -207,6 +207,12 @@ String ASTTableIdentifier::getDatabaseName() const
     else return {};
 }
 
+String ASTTableIdentifier::getTableName() const
```

Comment:
How is this function different from `getTable`? Why do we need it?

Reply:
Yes, no need, I'll remove it.